### ISSUE TITLE
Importing NUTCRACKER into VIXEN

### ISSUE THREAD

**Issue by keebler125** [smeighan/nutcracker#31]
when i imported my files into vixen, the FIRST file i have to remove several frames at the beginning manually

**Comment by smeighan** [comment]
Please always include your target, user and effect name.
OR
just paste the url from yoru animated gif. The url has all 3 bits of info.

it would help if you could make a video of this? I am not familiar with vixen so i would like to undertand the issue.

**Comment by gsxrgirl7** [comment]
Sean it does the same with mine - in vixen as well, where once they are pasted it has like a string or two or a blurp of lines, i dont necessarily remember it doing that in LOR on the imports, or if it did, it was not near as noticeable.  i do have a video on my iphone from vixen if you would like to see how it makes it jump abruptly.  where would you like me to send it to or email it to?  Here are a couple of links to my gifs that it does it to...
http://meighan.net/nutcracker/effects/workspaces/419/JULIE_TESTER~RTOG_SPARKLESPIRAL1.gif

**Comment by gsxrgirl7** [comment]
this one too
http://meighan.net/nutcracker/effects/workspaces/419/JULIE_TESTER~CANDYCANE3_1.gif
It does others, but was pretty noticeable in these two

**Comment by kgustafson** [comment]
gsxrgirl7 and keebler,  need to know if you still have issues with Vixen from the "Kurt Project" export.  I did a lot of reverse engineering into the crazy way that Vixen stores the vix file (trust me, it is crazy--but elegant in a way) and I am not seeing any issues now when I run my output on Vixen (which happens to be the sequencer I use the most).  

Kurt

**Comment by keebler125** [comment]
Where do i get the fix for vixen? Ill try it

**Comment by gsxrgirl7** [comment]
Ok I will have to test it and see.  Thanks for the update and I will check it today and let you know!  

Julie

**Comment by kgustafson** [comment]
Just download the from github to your local.  If you are using Meighan.net, the fixes are already in (although Meighan.net tends to lag github by a few days -- which is because we do extensive testing before committing all code to the 'production' server).

**Comment by gsxrgirl7** [comment]
Sorry I have not been able to respond yet.  Things got really hectic the last several days.  I will try to test it tonight as I had hoped to the other day.  And let you know.  
Sorry for the delay. 
Julie

**Comment by keebler125** [comment]
OK, sorry got REAL hectic here lately,

Will d/l the fix and apply it and hopefully test it tonight.

Thank you

K

**Comment by keebler125** [comment]
Just checked it in VIXEN2.14

Works fine (thank you)

That leaves just one more problem

The color correction.

As I told sean, the colors that I create in nutcracker are WAY of fin vixen.

I am using ws2801 pixels

I showed sean a chart I made in excel showing the colors that I wanted to use in nutcracker and the way I had to modify them to (with the new hex codes) to make them work in vixen.

Sean said there might have been a way to adjust the intensity in nutcracker, but I have no idea how to do that.

Thank you again

Ken
